models/public/conv-deconvnet/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  conv-deconvnet model is used for classification. For details see paper <https://https://github.com/yalaudah/facies_classification_benchmark>.
+task_type: classification
+files:
+  - name: conv-deconvnet.onnx
+    size: 336287691
+    sha256: a62ac0587f554e5e56169cacb911d70d2145e3e41b4a4cead605a2b094910893
+    source: http
```

Comment:
![image](https://user-images.githubusercontent.com/25801568/101175315-13ab1a80-3656-11eb-85a3-add56cdba7c3.png)

Reply:
At the moment, the model has not been published, and it is not yet clear whether it will be ONNX or pth. There was a conversation about putting it on - https://download.01.org/ For now, ONNX and pth files are on the local network.

Reply:
For test purpose you may try DropBox: https://github.com/dkurt/open_model_zoo/blob/dk/mri_reconstruction/models/public/hybdrid_cs_model_mri/model.yml